src/mcts/search.h

```diff
@@ -196,6 +197,22 @@ class Search {
   friend class SearchWorker;
 };
 
+// Helper to score and sort terminals for preferred moves.
+class Search::TerminalScore {
+ public:
+  EdgeAndNode edge;
+
+  TerminalScore(EdgeAndNode edge);
+  bool operator<(const TerminalScore& other) const;
+  int GetMate() const { return std::copysign((plies_ + 1) / 2, score_); }
+
+ private:
+  float score_ = 0.0f;
+  int plies_ = 0;
+
+  static int CalcPlies(Node* terminal);
```

Comment:
There's usually no need for private static function (unless it accesses static private member variables, which in this case it doesn't). A function in anonymous namespace in implementation .cc file is better.

Reply:
Renamed `CountPliesToTrueTerminal` and moved to anonymous namespace with added comments.